config/src/config/json_rpc_config.rs

```diff
@@ -42,3 +45,28 @@ impl JsonRpcConfig {
         self.address.set_port(utils::get_available_port());
     }
 }
+
+/// This API is experimental and subject to change
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct StreamConfig {
+    pub enabled: bool,
+    pub subscription_fetch_size: u64,
+    pub send_queue_size: usize,
+    pub poll_interval_ms: u64,
+}
```

Comment:
we should really point the user at the docs for this service or we just contribute to the general uselessness of these.... for example, take a look at the amazing metrics config that controls a useless framework but sounds really important.